src/Bundle/DependencyInjection/Compiler/TwigPass.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ResourceBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+final class TwigPass implements CompilerPassInterface
```

Comment:
Let's add note, that this class will be removed in Resource 2.0

Reply:
done

Reply:
Shouldn't we deprecate this class and inform about removing setting twig as public service in the UPGRADE file?

Reply:
I think deprecate it should be landed with Sylius resource controller split (next release). Cause we need this usage for this release and can not remove using it.